Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/tests-dashboard/test-dashboard.pipe.ts

```diff
@@ -1,17 +1,21 @@
-﻿import { Pipe, PipeTransform } from '@angular/core'
-@Pipe({
+﻿import { Pipe, PipeTransform } from '@angular/core';
+import { Test } from '../tests.model';
 
+@Pipe({
     name: 'filter'
-
 })
 export class FilterPipe implements PipeTransform {
 
-    transform(value: any, term: any): any {
-        if (term === undefined)
-            return value;
-        return value.filter(function (x: any) {
-
-            return (x.testName.toLowerCase().includes(term.toLowerCase()) || x.link.toLowerCase().includes(term.toLowerCase()));
-        })
+    transform(allTests: Test[], searchedTest: string) : any {
+        if (searchedTest === undefined)
```

Comment:
if (searchedTest === undefined) instead of this, you can write if(!searchedTest). it will check for empty, undefined and null. make necessary changes required.